hyperframe/tabulate.py

```diff
@@ -1,9 +1,11 @@
+from typing import Union
+
 import pandas as pd
 from bs4 import NavigableString, Tag
 
 # A path to some content within a hyper structure
 # e.g (div, 0, div, 2, p, text)
-HYPER_PATH = tuple[str | int, ...]
+HYPER_PATH = tuple[Union[str, int], ...]
```

Comment:
Should we rename this to `HyperPath`?

Reply:
sure 8a428db8ae48eced117cf8af8a51e69a8a4661c0